assets/src/components/FeaturesTable.js

```diff
@@ -0,0 +1,446 @@
+/**
+ * @module components/FeaturesTable.js
+ * @name FeaturesTable
+ * @copyright 2024 3Liz
+ * @author DOUCHIN Michaël
+ * @license MPL-2.0
+ */
+
+import { html, render } from 'lit-html';
+import { mainLizmap, mainEventDispatcher } from '../modules/Globals.js';
+
+/**
+ * @class
+ * @name FeaturesTable
+ * @augments HTMLElement
+ */
+export default class FeaturesTable extends HTMLElement {
+
+    constructor() {
+        super();
+
+        // Random element id
+        this.id = window.crypto.randomUUID();
+
+        // Layer name
+        this.layerTitle = this.getAttribute('layerTitle') || 'Features table: error';
+
+        // Layer id
+        this.layerId = this.getAttribute('layerId');
+
+        // Error text
+        this.error = null;
+
+        // Get the layer name & configuration
+        const layerName = lizMap.getLayerConfigById(this.layerId)?.[0];
+        let layerConfig = lizMap.getLayerConfigById(this.layerId)?.[1];
+        this.layerConfig = layerConfig;
+
+        // Primary key field
+        this.uniqueField = this.getAttribute('uniqueField');
+
+        // Expression filter
+        this.expressionFilter = this.getAttribute('expressionFilter');
+
+        // Get the geometry or NetworkError
+        this.withGeometry = (this.getAttribute('withGeometry') !== null) ? this.getAttribute('withGeometry') : 0;
+
+        // Sorting attribute and direction
+        this.sortingField = this.getAttribute('sortingField');
+        const sortingOrder = this.getAttribute('sortingOrder');
+        this.sortingOrder = (sortingOrder !== null && ['asc', 'desc'].includes(sortingOrder)) ? this.sortingField : 'asc';
+
+        // open popup ?
+        this.openPopup = (layerConfig && layerConfig.popup.toLowerCase() === 'true');
```

Comment:
```suggestion
        this.openPopup = (layerConfig && layerConfig.popup);
```
In `LayerConfig`, `popup` is a boolean